.travis.yml

```diff
@@ -1,13 +1,10 @@
 language: go
 go:
-- 1.8.x
-- 1.9.x
-- 1.10.x
 - 1.11.x
 branches:
   only:
   - master
-install: make bootstrap
+install: make ci
```

Comment:
You can just remove this line entirely. Travis will run make with no args automatically, which triggers make ci